### PR TITLE
LINK-2008, LINK-2047 | improvements to external user event form

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Move publishing notification position
Hide type section from external user in create event form 
Hide registration price groups from external user event form
Change submit button text for external users
User publicEventSchema as base for the external user schema 
Rename "Tallenna ajankohta" button as "Tallenna ajankohta"

## Closes
[LINK-2008](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2008)
[LINK-2047](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2047)

[LINK-2008]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2008?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LINK-2047]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/event/eventForm/EventForm.tsx [hunk 1]
```diff
@@ -430,13 +432,51 @@ const EventForm: React.FC<EventFormProps> = ({
...
 
-              <Section title={t('event.form.sections.type')}>
-                <TypeSection
-                  isEditingAllowed={isEditingAllowed}
-                  isExternalUser={isExternalUser}
-                  savedEvent={event}
-                />
-              </Section>
+              <Notification
+                style={{ marginBottom: 'var(--spacing-m)' }}
```
Comment (harriris-vincit) [inline src/domain/event/eventForm/EventForm.tsx:436]: Maybe use a class name instead of inline style?

**Review by harriris-vincit — APPROVED** [review]
One remark about inline style being used instead of a class name, but otherwise LGTM.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr399-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr399-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**